Tools/DeployApps/BaasClient.cs

```diff
@@ -556,6 +561,14 @@ private async Task<(BaasApp App, string MongoServiceId)> CreateAppCore(string na
                 development_mode_enabled = true,
             });
 
+            // Retrieve feature flags
+            // var features = await GetPrivateAsync<BsonDocument>($"groups/{_groupId}/apps/{app}/features");
+            // ENABLE LEGACY MIXED SUPPORT
+            // await PostPrivateAsync<BsonDocument>($"features/legacy_mixed_type", new { app_ids = new []{$"{appId}"}, action = "enable"} );
+            // await PostPrivateAsync<BsonDocument>($"features/bypass_legacy_mixed_type", new { app_ids = new []{$"{appId}"}, action = "disable"} );
+            // DISABLE LEGACY MIXED SUPPORT - Default on latest test server - https://github.com/10gen/baas/blob/master/etc/configs/test_config.json#L303
+            // await PostPrivateAsync<BsonDocument>($"features/legacy_mixed_type", new { app_ids = new []{$"{appId}"}, action = "disable"} );
+            // await PostPrivateAsync<BsonDocument>($"features/bypass_legacy_mixed_type", new { app_ids = new []{$"{appId}"}, action = "enable"} );
```

Comment:
We probably don't need this. While it's likely useful information, by the time we need it again, no one will remember it's here.